examples/05_stable_diffusion/scripts/compile.py

```diff
@@ -57,6 +54,13 @@ def compile_diffusers(
         torch_dtype=torch.float16,
     ).to("cuda")
 
+    assert (
+        width % 64 == 0
+    ), "Width must be divisible by 64, otherwise, the compilation process will fail."
+    assert (
```

Comment:
Shorter version:
```
    assert (
        height % 64 == 0 and width % 64 == 0
    ), "Height and Width must be divisible by 64, otherwise, the compilation process will fail."
```

Reply:
Point of this split was to tell the user which one was incorrect individually, as this process only happens once and is not slowing down the execution.

If you want me to do it like mentioned, please reply.

Reply:
I, as a user, would prefer to get more information about what is wrong with my inputs instead of getting errors one by one. `height` and `width` are properties of one object - the image. Just tell me what are `height` and `width` restrictions in ONE error message.

Another question: "multiples of 64" vs "divisible by 64" - looks like the existing code uses both phraseologies

Reply:
Updated